packages/language-server/src/lib/ast/block.ts

```diff
@@ -136,7 +136,7 @@ export function getFieldTypesFromCurrentBlock(schema: PrismaSchema, block: Block
   const fieldTypeNames: Record<string, string> = {}
 
   let reachedStartLine = false
-  for (const [document, lineIndex, line] of schema.iterLines()) {
+  for (const { document, lineIndex, text: line } of schema.iterLines()) {
```

Comment:
It looks like text / line isn't used here?
```suggestion
  for (const { document, lineIndex } of schema.iterLines()) {
```

Reply:
It is, on the line 150, github just does not show the diff that far away